docs/api-docs/in-depth-guides/rail-integration-guide.mdx

```diff
@@ -1,100 +1,154 @@
+| Rail Unloaded  | `container.transport.rail_unloaded` | The container is unloaded from a railcar. |
+
+There's also a set of events that are triggered when the status of the container at the destination rail terminal changes.  For containers without rail, they would have been triggered at the ocean terminal.
 
-#### Rail Milestones (aka rail events)
+| Milestone | Webhook Notification | Description |
+|----------------|----------------------|-------------|
+| Available for Pickup | `container.transport.available`     | (Coming Soon) The container is available for pickup at the rail terminal. |
+| Not Available  | `container.transport.not_available` | (Coming Soon) The container is not available for pickup at the rail terminal. |
+| Holds and Fees Updated | `container.transport.holds_and_fees_updated` | (Coming Soon) The holds and fees for the container at the rail terminal have been changed, but not fully cleared. |
+| Full Out       | `container.transport.full_out`      | The full container leaves the rail terminal. |
+| Empty In       | `container.transport.empty_in`      | The empty container is returned to the terminal. |
 
-Terminal49 provides a series of webhook notifications to keep you updated on key milestones in a container’s rail journey. These notifications allow you to integrate near real-time tracking data directly into your applications. Here’s a list of the supported rail events:
+Finally, we plan to provide webhook notifications for when estimated rail arrival times change, but this is still under development.
 
 | Milestone | Webhook Notification | Description |
 |----------------|----------------------|-------------|
-| Rail Loaded    | `container.transport.rail_loaded`   | Notification when the container is loaded onto a railcar. |
-| Rail Departed  | `container.transport.rail_departed` | Notification when the container departs from the origin terminal (usually port of discharge). |
-| Rail Arrived   | `container.transport.rail_arrived`  | Notification when the container arrives at the destination terminal. |
-| Rail Unloaded  | `container.transport.rail_unloaded` | Notification when the container is unloaded from a railcar. |
-| Full Out       | `container.transport.full_out`      | Update when the full container leaves the rail terminal. |
-| Empty In       | `container.transport.empty_in`      | Notification when the empty container is returned to the terminal. |
 | Estimated Rail Arrival | `container.estimated.rail_arrival` | (Coming soon) Estimated time of arrival for the container at the rail terminal. |
 
-Integrate these notifications by subscribing to the webhooks and handling the incoming data to update your systems. 
+Integrate these notifications by subscribing to the webhooks and handling the incoming data to update your systems.
+
+> Note: We do not have webhook notifications planned for Train Passing, Rail Interchange Received, or Rail Interchange Delivered.  However, you can still see them in the dashboard.
```

Comment:
I'd remove this - let's see what people ask for and it's quite likely we might add some specific Rail Interchange notifications

Reply:
Removed